lib/plugins/aws/package/compile/functions/index.js

```diff
@@ -719,6 +746,48 @@ class AwsCompileFunctions {
       Value: 'Value',
     };
   }
+
+  addFileContentsToHashes(filePath, hashes) {
+    // Read the file in chunks and add them to the hash (saves memory and performance)
+    return new BbPromise((cb, reject) => {
+      const readStream = fs.createReadStream(filePath);
+      readStream
+        .on('data', chunk => {
+          hashes.forEach(hash => {
+            hash.write(chunk);
+          });
+        })
+        .on('close', () => {
+          cb();
+        })
+        .on('error', error => {
+          reject(new this.serverless.classes.Error(`Could not file content to hash: ${error}`));
```

Comment:
I think it should stay as regular error, as it can be only an issue in how framework handles things and not issue with user setup, isn't it?